deployment/run_server

```diff
@@ -1,10 +1,34 @@
 #!/bin/bash -xe
 
 fc-cache -v /usr/share/fonts/user
+set -e
+
+BUILD_DIR=/tmp/plotly-graph-exporter-build
+URL_REGEXP='(https?|ftp|file)://[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]'
+ARGS=''
+
+while [[ $# -gt 0 ]]
+  do
+    key="$1"
+    value="$2"
+    # Check for valid url and update args:
+    if [[ $key = "--plotlyJS" ]] && [[ $value =~ $URL_REGEXP ]]; then
+
+      mkdir -p $BUILD_DIR
+      wget --tries 40 --retry-connrefused -O $BUILD_DIR/plotly-bundle.js $value
```

Comment:
using wget instead of `curl` because it has `--retry-connrefused` option. Curl also has this option in newer versions, which requires an upgrade. This works as it is.

I noticed that it takes ~10 retries to fetch the bundle on buildly , so keeping it to 40 to be on the safer side.

Reply:
Can you explain the consequences if this download fails?

Reply:
It fails silently. imageserver will not start, but the other onprem services go on as usual.

This is because we report streambed's `/ping` as ready-state. Maybe we can add imageserver's `ping` as well? i.e. if we want the service to error out on imageserver start failure.